homeassistant/components/nissan_leaf/switch.py

```diff
@@ -59,6 +56,9 @@ def is_on(self):
         if await self.car.async_set_climate(False):
             self.car.data[DATA_CLIMATE] = False
 
+    # @MartinHjelmare would like removed - think provides nice UI feedback
+    # for switch.
+    # Think VolvoOnCall component hase different icons for the switches.
```

Comment:
VolvoOnCall doesn't have different icons for the switches. With Lovelace frontend it's super easy to customize this anyway.

Reply:
Removed icons for climate control in commit https://github.com/home-assistant/home-assistant/pull/21145/commits/d7f9e83f9514b3aa5de328f38e0c5d63f76d6435

Reply:
Removed icon with commit https://github.com/home-assistant/home-assistant/pull/21145/commits/eb04b4d04661b965f51008510313cc44ca37149c